src/components/extractFields.js

```diff
@@ -38,6 +38,14 @@ function getLon(properties) {
   }
 }
 
+function getBoundingBox(properties) {
+  if (properties['lbl:bbox']) {
```

Comment:
I think you're right about the various conditions and the code will work fine for this scenario. However, it's not a great convention and doesn't make the true intent of this statement clear. I recall discussing this with the team a long time ago and coming to an agreement to use hasOwnProperty or isSet checks for these cases. It's not a big deal, but I think it makes the code a bit easier to understand.